region/norfair/crocomire.json

```diff
@@ -541,8 +545,13 @@
           "sparking": {
             "runways": [
               {
-                "length": 31,
+                "length": 4,
                 "requires": null
+              },
+              {
+                "length": 31,
+                "requires": null,
+                "usableComingIn": false
```

Comment:
It would be usableComingIn with Gravity Suit actually, and some acid frames, but since there's no charge strats here it doesn't really matter.

Reply:
You can even use it without gravity many times but didn't want to add a lot of acid junk for no use.

Reply:
Yeah I agree with that.